src/font.rs

```diff
@@ -517,6 +785,13 @@ impl<'a> Font<'a> {
             None => None,
         }
     }
+
+    pub fn lookup_glyph_id(&self, code_point: u16) -> Option<u16> {
```

Comment:
It's probably better for code_point to be u32 here (char would also be a possibility, but I think it's possible to map surrogates), to handle the future case where format 12 is handled.